tests/testsuite.src/run_extensions.at

```diff
@@ -123,95 +123,6 @@ AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
 
 AT_CLEANUP
 
-
-AT_SETUP([HP COBOL octal literals])
```

Comment:
Those are runtime tests which actually need to be executed to verify that the translation is done correctly. Therefore those need to stay in here (and the one necessary runtime check with [[]] should be added below.